### PR TITLE
Add page Debug tracing

### PR DESCRIPTION
**Pull request by blukat29**
## Proposed changes

- Technical guide about getting debug traces

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Minor Issues and Typos
- [x] Major Content Contribution
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to reach out. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/kaiachain/kaia-docs/blob/main/CONTRIBUTING.md)
- [x] I have read the [CLA](https://gist.github.com/kaiachain-dev/bbf65cc330275c057463c4c94ce787a6) and signed by comment ```I have read the CLA Document and I hereby sign the CLA``` in first time contribute
- [x] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large content contribution, kick off the discussion by explaining why you would suggest the content contribution, etc...

### CODE REVIEW

File: docs/learn/computation/debug-trace.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+When an full-mode node serves the debug trace APIs it may undergo historical state regeneration. Suppose you requested to trace a block. The node has to execute the block from the parent block's final state. But a full-mode node stores block states only so often (every other 128 blocks by default), so the node re-exectues blocks from the nearest saved state. At worst case tracing one block takes executing 127 blocks. Therefore a full-mode node may respond to debug trace APIs slowly. If you are going to requeat a large number, you must use an archive-mode nodes.
+
+## Using chaindatafetcher
+
+Chaindatafetcher (CDF) is a unique feature of Kaia where block processing results are published into Kafka queue to simplify building downstream data services. For instance a block explorer can utilize CDF to populate its database.
+
+### Connecting to Kafka
+
+Prepare a Kafka cluster first. Below is an example docker compose but you can use any Kafka installation with any topic. However, only the ____ authentication is currently supported.
```
[the remark this answers is not in the document]
Comment (blukat29) [reply docs/learn/computation/debug-trace.md]: I see that authentication isn't supported.

**Comment by blukat29** [comment]
@ian0371 PTAL

File: docs/learn/computation/debug-tracing.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+- Receive into kafka via chaindatafetcher from a node as the node syncs blocks.
+- Use block explorers or indexer services, which is outside scope of this document.
+
+## Using debug APIs
+
+The go-to method is to calling the `debug_trace*` JSON-RPC APIs.
+
+### Tracer types
+
+To get traces with the debug API, you first need to decide what kind of information you want to learn. Most commonly, you'll want to use `callTracer` to understand the interaction between contracts i.e. internal transactions. There are other tracer types like `prestateTracer` and structLogger that allow you to debug the same transaction from multiple angles. Below lists some frequently used tracers.
```
Comment (ian0371) [inline docs/learn/computation/debug-tracing.md:18]:
```suggestion
To get traces with the debug API, you first need to decide what kind of information you want to learn. Most commonly, you'll want to use `callTracer` to understand the interaction between contracts i.e. internal transactions. There are other tracer types like `prestateTracer` and `structLogger` that allow you to debug the same transaction from multiple angles. Below lists some frequently used tracers.
```

**Review by ian0371 — APPROVED** [review]

File: docs/learn/computation/debug-tracing.md (hunk 2 above)
Comment (blukat29) [reply docs/learn/computation/debug-tracing.md]: Fixed [c8388ff](https://github.com/kaiachain/kaia-docs/pull/44/commits/c8388ffe6368c1325433afcdcdf44e9bd361f337)

**Review by scott-klaytn — APPROVED** [review]

**Pull request merged**